source/falaise/snemo/services/dead_cells.cc

```diff
@@ -29,13 +29,25 @@ namespace snemo {
   int dead_cells_svc::LoadCells(std::string deadcells_filename) {
     // Creates a vector of _ONLY_ Bad cells (e.g reading from a DB or from a file like now);
     // cells not included in the vector are considered to be Good;
-    // the vector here created can be independently accessed
+    // the vector here created can be also accessed (size, loops).
+
     try {
       std::ifstream infile(deadcells_filename);
       int side, layer, column, status;
+      // I assume here that the `status` is saved as `int`, therefore the switch
+      
       while (infile >> side >> layer >> column >> status){
+	cell_status cs = cell_status::good;
+	switch(status) {
+	case 1: cs = cell_status::dead; break;
+	case 2: cs = cell_status::cathode_ground_top; break;
+	case 3: cs = cell_status::cathode_ground_bottom; break;
+	case 4: cs = cell_status::cathode_ground_both; break;
+	case 5: cs = cell_status::cathode_cathode; break;
+	case 6: cs = cell_status::other; break;
+	}  
```

Comment:
Do we want any error handling in the case the read `int` is outside 0-6? At the moment, the status will be set to `cell_status::good`, which I think is o.k., but just wondered if an exception should be thrown, or non-zero returned?